db/data_migration/20160202102043_generate_content_ids_on_classifications.rb

```diff
@@ -0,0 +1,5 @@
+require 'securerandom'
+
+Classification.where(content_id: nil).each do |classification|
+  classification.update_column(:content_id, SecureRandom.uuid)
```

Comment:
After a conversation in person, the consensus is that data migrations are a better way to do this than a rake task...